src/__tests__/__snapshots__/index.js.snap

```diff
@@ -7,11 +7,17 @@ exports[`allows you to pass custom props that are allowed 1`] = `
 }
 
 <div
-  class="css-1r7lsit"
+  class="css-1r7lsit glamorous(mockConstructor)"
   id="hello"
 />
 `;
 
+exports[`allows you to specify a displayName 1`] = `
+<div
+  class="css-nil HiThere"
```

Comment:
Here's the real user-experience win right here. If the component has a `displayName` then we get that right here. And if folks are using the babel plugin, then that should work great.